contracts/bonding/IBondingManager.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.4.11;
+
+/*
+ * @title Interface for BondingManager
+ * TODO: switch to interface type
+ */
+contract IBondingManager {
+    // External functions
+    function transcoder(uint8 _blockRewardCut, uint8 _feeShare, uint256 _pricePerSegment) external returns (bool);
+    function resignAsTranscoder() external returns (bool);
+    function bond(uint _amount, address _to) external returns (bool);
+    function unbond() external returns (bool);
+    function withdraw() external returns (bool);
+    function reward() external returns (bool);
+
+    // Public functions
+    function setActiveTranscoders() public returns (bool);
```

Comment:
Hmm, what's unique about `setActiveTranscoders()` that would make it the only publicly scoped state modifying transaction across all three interfaces? Is it the only one that needs to be invoked by another manager?